### PR TITLE
Typescript app

### PR DESCRIPTION
**Pull request by beausmith**
Ok! First PR here. Take it for a ride. 

Looking forward to your comments and questions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,39 @@
 {
-  "name": "bmd",
+  "name": "bmd-typescript",
```
Comment (benadida) [inline package.json]: do we need to call it `-typescript`?

File: public/index.html [hunk 2]
```diff
@@ -19,13 +22,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>VotingWorks Ballot Marking Device</title>
+    <title>React App</title>
```
Comment (benadida) [inline public/index.html]: couple small things lost like the title of the page, but not a big deal, we can fix later.

File: package.json (hunk 1 above)
Comment (beausmith) [reply package.json]: pushed fix.

File: public/index.html (hunk 2 above)
Comment (beausmith) [reply public/index.html]: pushed fix.

**Pull request merged**